src/reducers/BackendModeSignedInStatus.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+import Cookies from 'js-cookie';
+// import {fetchWithTimeout} from '../util/FetchWithTimeout';
+
+export const SERVER_BASE_URL = process.env.NODE_ENV === 'development'
+  ? "http://localhost:5000/" : "https://yaas.azurewebsites.net/";
+
+export const BACKEND_MODE_SIGNED_IN_STATUS = {
+  USER_SIGNED_IN: 'USER_SIGNED_IN',
+  USER_SIGNED_OUT: 'USER_SIGNED_OUT',
+  LOCAL_STORAGE: 'LOCAL_STORAGE',
+}
+
+export const ACCESS_TOKEN_COOKIE_KEY = 'access_token';
+
+export async function loginBackend(name, email, token) {
+  await axios.post(
+    SERVER_BASE_URL + "register_user",
+    {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json', 'Set-Cookie': `token=${token}` },
+      body: JSON.stringify({ name, email, token })
+    },
+    { withCredentials: true },
+  );
+  return true;
+};
+
+export const INITIAL_LOCAL_STORAGE_BACKEND_MODE_LOCAL_STORAGE_KEY = 'initialLocalStorageBackendMode';
+
+export const getUserSignedInStatus = () => {
+  const token = Cookies.get(ACCESS_TOKEN_COOKIE_KEY);
+  if (token) {
+    return BACKEND_MODE_SIGNED_IN_STATUS.USER_SIGNED_IN;
+    // TODO
+    // try {
+    //   const { ok } = fetchWithTimeout(
+    //     SERVER_BASE_URL + `auth`,
+    //     { headers: new Headers({ 'Set-Cookie': `token=${token}` }) },
+    //   );
+    //   if (ok) { return BACKEND_MODE_SIGNED_IN_STATUS.USER_SIGNED_IN; }
+    // } catch (e) { console.log(e); }
```

Comment:
I was trying to use the auth testing API as like an initial check that the stored token is valid and the backend service is running, but I kept getting errors from the endpoint. I think that this is a worthwhile check to do on loading the app, so can if anyone has any ideas on why the request wasn't working ...

Reply:
I'll test this once we merge this

Reply:
I think it's a cross origin thing. I'l check while looking at it tomorrow as well